notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<p>What is the one variable of the ARIMA_PLUS univariate model?</p><p>What are the multiple variables of the Vertex Forecasting model?</p><p>"more signal" = more input variables?</p><p>"without knowing these factors" -&gt; maybe "without knowing the values of these factors"? or "without knowing the impact / significance / importance of these factors"?</p><p> </p><p>Is it possible to display the dataset (or part of it) to help the user understand the problem we're solving here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p><strong>What is the one variable of the ARIMA_PLUS univariate model?</strong></p><p>Modified to state more clearly we're forecasting sales.</p><p> </p><p><strong>What are the multiple variables of the Vertex Forecasting model?</strong></p><p>A few variables are stated already: "advertisements, holidays, and locations". </p><p> </p><p><strong>"more signal" = more input variables?</strong></p><p>Rephrased to make it more clear we'll be adding in the factors sales are dependent on for the multivariate model.</p><p> </p><p><strong>"without knowing these factors" -&gt; maybe "without knowing the values of these factors"? or "without knowing the impact / significance / importance of these factors"?</strong></p><p>I rephrased to "information about these factors". "The value" sounded confusing since typically we don't think of advertisement or holiday effects as a value even if that's what the model sees. The model learns the importance of each factor in the form of feature weights, this is not information we would be able to provide.</p><p> </p><p><strong>Is it possible to display the dataset (or part of it) to help the user understand the problem we're solving here?</strong></p><p>This occurs later in the notebook since we haven't created the data at this point.</p>